continuous_integration/install.sh

```diff
@@ -94,6 +97,13 @@ elif [[ "$DISTRIB" == "conda" ]]; then
     if [ -n "$NIBABEL_VERSION" ]; then
         pip install nibabel=="$NIBABEL_VERSION"
     fi
+    if [["$PYTHON_VERSION" == "2.6"]]; then
```

Comment:
something like "with python 2.6 we install the minimum version supported for patsy and nilearn otherwise the latest?"